AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Shared/EditorTemplates/Location.cshtml

```diff
@@ -2,14 +2,14 @@
 <div>
     <input type="hidden" asp-for="Id" />
     <div class="form-group">
-        <label asp-for="Address1" class="col-md-2 control-label"></label>
+        <label asp-for="Address1" class="col-md-2 control-label">Address Line 1</label>

```

Comment:
I think these can be affected on AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Shared/LocationDisplayViewModel.cs